salt/utils/event.py

```diff
@@ -328,7 +373,7 @@ def _get_event(self, wait, tag, pending_tags):
 
         return None
 
-    def get_event(self, wait=5, tag='', full=False, use_pending=False, pending_tags=None):
```

Comment:
My thought is to save existing `self.pending_tags` and then add the passed in `pending_tags` onto it - so they are preserved, and then restore `self.pending_tags` afterwards.